src/pay/payment_consumer.py

```diff
@@ -237,7 +237,12 @@ def create_stats_dict(self, nb_failed, nb_unknown, payment_cycle, payment_logs,
         stats_dict['nb_delegators'] = n_d_type
         stats_dict['pay_xfer_fee'] = 1 if self.delegator_pays_xfer_fee else 0
         stats_dict['pay_ra_fee'] = 1 if self.delegator_pays_ra_fee else 0
-        stats_dict['rewards_type'] = "I" if self.rewards_type.isIdeal() else "A"
+        if self.rewards_type.isIdeal():
+            stats_dict['rewards_type'] = "I"
+        elif self.rewards_type.isActual():
```

Comment:
@nicolasochem don't we need an `else:  stats_dict['rewards_type'] = "A"` at the end as the "default value"?

Reply:
Yes, it is better to have a default value here. I added it in d45441d. Thank you